components/Form/index.js

```diff
@@ -0,0 +1,49 @@
+import { FormContainer, FormButton } from "./styles-form";
+import { Heading } from "./styles-form";
+import { useRouter } from "next/router";
+
+export default function CreateRecipe() {
+  const router = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const recipe = Object.fromEntries(formData);
+
+    const response = await fetch(`/api/recipes`, {
+      method: "POST",
+      body: JSON.stringify(recipe),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    event.target.reset();
+    router.push("/");
```

Comment:
Even if you are not interested in the response of the HTTP, it's important to intercept any potential error:

```js
try {
  const response = await fetch(`/api/recipes`, {
      method: "POST",
      body: JSON.stringify(recipe),
      headers: {
        "Content-Type": "application/json",
      },
  });
  if (!response.ok) {
    window.alert('Problem creating the recipe!');
    return;
  }
  event.target.reset();
  router.push('/');
} catch(error) {
  window.alert('Problem creating the recipe!');
}
```